src/app/beer_garden/scheduler.py

```diff
@@ -530,6 +556,48 @@ def create_job(job: Job) -> Job:
     return job
 
 
+def create_jobs(jobs: JobDefinitionList) -> JobIDList:
+    """Create multiple new Jobs and add them to the scheduler.
+
+    Args:
+        jobs: A `JobDefinitionList` that holds the `Job`s to be added
+
+    Returns:
+        A `JobIDList` that holds the `id`s of the `Job`s that were added
+    """
+    jobs_list: Iterator[Job] = jobs.jobs
+    jobs_created_list: List[Job] = []
+    job_collection_name = "job"
+
+    # we want the addition of Jobs to obey ACID: either all will succeed or none
+    # will succeed; hence we skip using mongoengine and use the pymongo library
+    # directly
+
+    the_connection: pymongo.MongoClient = mongoengine.get_connection()
+    the_db: pymongo.database.Database = mongoengine.get_db()
+
+    session: ClientSession
+    with the_connection.start_session() as session:
+        job_collection: Collection = the_db.get_collection(job_collection_name)
+
+        for job in jobs_list:
+            mongo_obj: mongoengine.Document = from_brewtils(job)  # noqa
+            job_as_son: bson.BSON = mongo_obj.to_mongo()
```

Comment:
was this intended to be job_as_**b**son?

Reply:
Surprisingly, no it was not. That's how the library authors chose to name their functions.